packages/relay/src/runServer.ts

```diff
@@ -28,6 +28,8 @@ function error (err: string): never {
   process.exit(1)
 }
 
+const DEBUG_PROVIDER = true
```

Comment:
don't default it to true..
(maybe add it to config? this way it can be enabled on a real, running relayer.)

Reply:
Added to config